docs/getting_started.rst

```diff
@@ -139,8 +224,8 @@ Now from the fully-overlaid shell, you can run the soss instance:
     soss src/soss-ros1/examples/hello_ros.yaml
 
 
-The executable `soss` is given a `.yaml` configuration file to describe how messages should be passed
-through whichever middlewares (in this case, ROS1 and ROS2).
+In this command, the executable `soss` is given a `.yaml` configuration file to describe how messages should be passed
+among whichever middlewares (in this case, ROS1 and ROS2).
```

Comment:
```suggestion
through whichever middlewares (in this case, ROS1 and ROS2).
```

I think is much more clearer to understand the flow of the information to use `through` instead of `among`.

Reply:
I would say that through is just wrong in this case.